### PR TITLE
feat: `jsonIndentation` for better formatting

### PR DESCRIPTION
**Pull request by A7med3bdulBaset**
closes #1

### CODE REVIEW

File: lib/utils.ts [hunk 1]
```diff
@@ -43,7 +44,30 @@ export const configPath = configPathExpected || configPathRoot
...
+      `${chalk.cyan.bold(key)} is not set in ${chalk.yellow(configFileName)}`,
+      `By default it is set to ${chalk.cyan(defaultValue)}`,
+      `You can change it in ${chalk.yellow(configFileName)} or run ${chalk.cyan(
+        'linguify init'
+      )} again to configure missing config`
+    ].join('\n')
+  )
+
+  return defaultValue
+}
```
Comment (A7med3bdulBaset) [inline lib/utils.ts:70]: This utility to keep the dev up-to-date with the config updates when releasing new configurations like this.

Comment (A7med3bdulBaset) [reply lib/utils.ts]: This utility can replace `linguifyValidation()`

Comment (rawand-faraidun) [reply lib/utils.ts]: the validations in `linguifyValidation()` is not only for checking if it is set or not, so it is still needed

**Comment by rawand-faraidun** [comment]
Thank you for the work, I have to add some slight changes so I will squash it into dev and work on it

**Pull request merged**